lib/routes/xiaohongshu/notes.ts

```diff
@@ -15,25 +18,125 @@ export const route: Route = {
     handler,
     example: '/xiaohongshu/user/52d8c541b4c4d60e6c867480/notes/fulltext',
     features: {
+        requireConfig: [
+            {
+                name: 'XIAOHONGSHU_COOKIE',
+                optional: true,
+                description: '小红书 cookie 值，可在浏览器控制台通过`document.cookie`获取。',
+            },
+        ],
         antiCrawler: true,
         requirePuppeteer: true,
     },
     parameters: {
         user_id: 'user id, length 24 characters',
+        fulltext: {
+            description: '是否获取全文',
+            default: '',
+        },
     },
 };
 
 async function handler(ctx) {
     const userId = ctx.req.param('user_id');
     const url = `https://www.xiaohongshu.com/user/profile/${userId}`;
 
-    const { user, notes } = await getNotes(url, cache);
+    if (config.xiaohongshu.cookie && ctx.req.param('fulltext')) {
+        const user = await getUser(url, config.xiaohongshu.cookie);
+        const notes = await renderNotesFulltext(user.notes, url);
+        return {
+            title: `${user.userPageData.basicInfo.nickname} - 笔记 • 小红书 / RED`,
+            description: user.userPageData.basicInfo.desc,
+            image: user.userPageData.basicInfo.imageb || user.userPageData.basicInfo.images,
+            link: url,
+            item: notes,
+        };
+    } else {
+        const { user, notes } = await getNotes(url, cache);
+        return {
+            title: `${user.nickname} - 笔记 • 小红书 / RED`,
+            description: formatText(user.desc),
+            image: user.imageb || user.images,
+            link: url,
+            item: notes.map((item) => formatNote(url, item)),
+        };
+    }
+}
+
+async function getUser(url, cookie) {
+    const res = await got(url, {
+        headers: {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36',
```

Comment:
RSSHub will use a randomised user agent of [Chrome on mac](https://github.com/DIYgod/RSSHub/blob/a41cfc79ec93186793bf5b98a1c46cbd9a03edd3/lib/config.ts#L440) by default. Does the site only work with this fixed version of Chrome on Windows?